zerver/openapi/zulip.yaml

```diff
@@ -4376,11 +4379,19 @@ paths:
                     properties:
                       result: {}
                       msg: {}
+                      ignored_parameters_unsupported:
+                        $ref: "#/components/schemas/IgnoredParametersUnsupported"
                       stream_id:
                         type: integer
                         description: |
                           The ID of the given stream.
-                    example: {"msg": "", "result": "success", "stream_id": 15}
+                    example:
+                      {
+                        "stream_id": 15,
+                        "ignored_parameters_unsupported": ["invalid_parameter"],
+                        "msg": "",
+                        "result": "success",
+                      }
```

Comment:
I don't think it really makes sense to change all of our example responses to show an unsupported parameter. That's not something that we expect users to see in practice often, and so it seems confusing for users to have it appear in every example. This seems much more like the sort of thing where we give a single example on the common error handling page, and that's it.

Does the `IgnoredParametersUnsupported` correctly mark this field as optional?